### PR TITLE
Xyl/fix mm and bmm

### PR DESCRIPTION
**Pull request by xue-yun-liang**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Description

Fix mm and bmm op retention issues

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: impl/ascend/device_configs.py [hunk 1]
```diff
@@ -908,16 +908,8 @@
...
-            atol=1e-3,
-            rtol=1e-3,
-            args=[
-                {
-                    "ins": ['input'],
-                    "shape": [Skip((16, 726, 32)),Skip((16, 100, 100)),Skip((9, 5, 5)),Skip((0, 12, 16)),Skip((4, 0, 6)),Skip((4, 9, 0)),Skip((5, 8, 13)),],
-                },
-            ]
-        ),
+        atol=3e-2,
```
Comment (hellozmz) [inline impl/ascend/device_configs.py:911]: 所有类型的数据都这么大的diff么

File: impl/ascend/functions/bmm.cpp [hunk 2]
```diff
@@ -12,6 +12,14 @@ namespace ascend {
 diopiError_t diopiBmm(diopiContextHandle_t ctx, diopiTensorHandle_t out, diopiConstTensorHandle_t input, diopiConstTensorHandle_t mat2) {
     AscendTensor inputCopy(input);
     AscendTensor mat2Copy(mat2);
+    AscendTensor outputCopy(out);
+    if (inputCopy.numel() == 0 || mat2Copy.numel() == 0) {
+        diopiScalar_t zero;
```
Comment (yangbofun) [inline impl/ascend/functions/bmm.cpp:17]: 这里有专门的constructDiopiScalar函数，下同

File: impl/ascend/device_configs.py (hunk 1 above)
Comment (xue-yun-liang) [reply impl/ascend/device_configs.py:911]: float32和float64精度只能到达0.01

File: impl/ascend/functions/bmm.cpp (hunk 2 above)
Comment (xue-yun-liang) [reply impl/ascend/functions/bmm.cpp]: 已修改

**Review by yangbofun — APPROVED** [review]

**Review by hellozmz — APPROVED** [review]

**Pull request merged**